### ISSUE TITLE
conditional display > Answer field does not work with answerValueSet

### ISSUE THREAD

**Issue by Rosnyni** [lhncbc/formbuilder-lhcforms#70]
Hello, 

The following JSON was created using the formbuilder : 

```
{
  "resourceType": "Questionnaire",
  "extension": [
    {
      "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-preferredTerminologyServer",
      "valueUrl": "https://hapi-fhir-server-dev-ext-k8s.eds.aphp.fr/fhir/"
    }
  ],
  "title": "New Form",
  "status": "draft",
  "item": [
    {
      "item": [
        {
          "type": "string",
          "linkId": "9191010295087",
          "text": "item that should be enabled only in certain circumstances, but it don't work so, there is no condition"
        }
      ],
      "type": "choice",
      "extension": [
        {
          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
          "valueCodeableConcept": {
            "coding": [
              {
                "system": "http://hl7.org/fhir/questionnaire-item-control",
                "code": "drop-down",
                "display": "Drop down"
              }
            ]
          }
        }
      ],
      "linkId": "2643956466117",
      "text": "choice item with answerValueSet",
      "answerValueSet": "https://aphp.fr/ig/fhir/sdc/ValueSet/aphp-sdc-aph-mat-ddg-vs"
    }
  ]
}
```

my apologize for using a local terminology serveur, but it works : 

![image](https://github.com/lhncbc/formbuilder-lhcforms/assets/61731606/34c7f593-1469-4226-ab60-be4afe4a7cbb)

However, it is not possible to file the conditional display > answer field of the 9191010295087 item, the list is desperatly empty : 

![image](https://github.com/lhncbc/formbuilder-lhcforms/assets/61731606/8fb35a5c-81e8-4f6d-8888-0ebb8d68c5ae)

**Comment by plynchnlm** [comment]
Thanks for reporting this.  This is being worked on now.

**Comment by akanduru** [comment]
This is fixed as of 9.2.6
